packages/tools/devtools/devtools-browser-extension/package.json

```diff
@@ -51,14 +56,14 @@
 		"cache-dir": "nyc/.cache",
 		"exclude": [
 			"src/test/**/*.*ts",
-			"dist/tsc/test/**/*.*js",
-			"dist/e2e-tests/**",
-			"dist/bundle/**"
+			"lib/tsc/test/**/*.*js",
+			"lib/e2e-tests/**",
+			"lib/bundle/**"
 		],
 		"exclude-after-remap": false,
 		"include": [
 			"src/**/*.*ts",
-			"dist/tsc/**/*.*js"
+			"lib/tsc/**/*.*js"
```

Comment:
I see most other projects didn't do this, so not sure it's correct.

Reply:
I put these back, since we're still testing only CJS for this package...  but in general, that's the right thing to do.